action.yml

```diff
@@ -26,6 +26,18 @@ inputs:
     description: 'Skip automated testing such as pytest'
     required: false
     default: false
+  prebuild-package-path:
+    description: 'Path to brebuild app zip'
+    required: false
+    default: ''
+  ignored-paths:
+    description: 'Path patterns to skip zip, separated by space'
+    required: false
+    default: ''
```

Comment:
Should it be an empty array or omitted?

Reply:
The string is split by spaces before passing to zip command so default `''` value is converted to empty array